msi_perkeyrgb/msi_keyboard.py

```diff
@@ -6,17 +6,402 @@
 
 AVAILABLE_MSI_KEYMAPS = [
              (["GE63", "GE73", "GS63", "GS73", "GS75", "GX63", "GT63", "GL63"],
-             {"fn": 0xf0, 38: 0x4, 56: 0x5, 54: 0x6, 40: 0x7, 26: 0x8, 41: 0x9, 42: 0xa, 43: 0xb, 31: 0xc, 44: 0xd, 45: 0xe, 46: 0xf, 58: 0x10, 57: 0x11, 32: 0x12, 33: 0x13, 24: 0x14, 27: 0x15, 39: 0x16, 28: 0x17, 30: 0x18, 55: 0x19, 25: 0x1a, 53: 0x1b, 29: 0x1c, 52: 0x1d, 10: 0x1e, 11: 0x1f, 12: 0x20, 13: 0x21, 14: 0x22, 15: 0x23, 16: 0x24, 17: 0x25, 18: 0x26, 19: 0x27, 67: 0x3a, 68: 0x3b, 69: 0x3c, 70: 0x3d, 71: 0x3e, 72: 0x3f, 36: 0x28, 51: 0x31, 94: 0x64, 9: 0x29, 22: 0x2a, 23: 0x2b, 65: 0x2c, 20: 0x2d, 21: 0x2e, 34: 0x2f, 35: 0x30, 47: 0x33, 48: 0x34, 49: 0x35, 59: 0x36, 60: 0x37, 61: 0x38, 66: 0x39, 37: 0xe0, 50: 0xe1, 64: 0xe2, 133: 0xe3, 105: 0xe4, 62: 0xe5, 108: 0xe6, 73: 0x40, 74: 0x41, 75: 0x42, 76: 0x43, 95: 0x44, 96: 0x45, 107: 0x46, 78: 0x47, 127: 0x48, 118: 0x49, 112: 0x4b, 119: 0x4c, 117: 0x4e, 114: 0x4f, 113: 0x50, 116: 0x51, 111: 0x52, 77: 0x53, 106: 0x54, 63: 0x55, 82: 0x56, 86: 0x57, 104: 0x58, 87: 0x59, 88: 0x5a, 89: 0x5b, 83: 0x5c, 84: 0x5d, 85: 0x5e, 79: 0x5f, 80: 0x60, 81: 0x61, 90: 0x62, 91: 0x63}),
+              {"fn": 
```

Comment:
I'm not in the Python world, but I feel adding new lines to AVAILABLE_MSI_KEYMAPS makes this file harder to read.

Reply:
Agreed. I'm not against formatting guidelines, but there are cases where there _shouldn't_ be respected, for the sake of readability. Same for https://github.com/Askannz/msi-perkeyrgb/pull/17/files#diff-9cb0b386da3a5f5240665e9b6fc8c6b3.